src/sync/index.js

```diff
@@ -171,8 +159,9 @@ async function sync(db) {
         if (_.isNil(chainBlockNum)) {
           logger.warn('chainBlockNum should not be null');
         } else if (startBlock >= chainBlockNum) {
-          pubsub.publish('OnSyncInfo', { OnSyncInfo: { syncBlockNum: currentBlockChainHeight, syncBlockTime: currentBlockTime, chainBlockNum } });
+          sendSyncInfo(currentBlockChainHeight, currentBlockTime, chainBlockNum);
```

Comment:
@dwalintukan @henryzbw this func will be executed anyway even though there is no new block come in. If local db is fully synced and chainBlockNum == max(blocks), startBlock = chainBlockNum +1 so the same info will keep emitting.

Reply:
to avoid this, change the if condition to `if numberOfIteration > 0`,

Reply:
fixed